web-client/public/js/graph.js

```diff
@@ -43,11 +43,8 @@ export var drawGraph = function (network, sliderController, nodeColoring) {
         return node.name.length * 12 + 5;
     };
 
-    /* If colorOptimal is false, then weighting is ignored,
+    /* If .colorOptimal is false, then weighting is ignored,
     and the lines are all drawn as if it was an unweighted sheet */
```

Comment:
This comment seems attached to the lines that you deleted; is there still reason for it to be here?